package.json

```diff
@@ -5,6 +5,7 @@
   "scripts": {
     "build:watch": "preconstruct watch",
     "build": "preconstruct build",
+    "tsc-all": "tsc && cd site && tsc",
```

Comment:
nit: other scripts use `:` as a delimiter. I don't mind your version, it's just noticeable that it uses a different "style" when I look at the whole list of `scripts`.